cl/scrapers/tests.py

```diff
@@ -438,3 +443,41 @@ def test_process_audio_file(self):
             msg="We should end up with the proper duration of about %s. "
             "Instead we got %s." % (expected_duration, measured_duration),
         )
+
+
+class BinaryTransformerExtractionTest(TestCase):
+    def setUp(self):
+        self.path = os.path.join(settings.MEDIA_ROOT, "test", "search")
+
+    def test_heartbeat(self):
+        """Can we start container and check sanity test?"""
+        base_url = "http://cl-binary-transformers-and-extractors:80"
+        response = requests.get(url=base_url, timeout=60).json()
```

Comment:
Tests should be fast, especially the heartbeat. We don't want to wait a minute if it's down. Give it a second, maybe two.

Reply:
√